lib/components/common/Header.tsx

```diff
@@ -1,67 +1,66 @@
-import { Box, Button, Divider, IconButton, Typography } from "@mui/material";
-import { vars } from "../../theme/variables";
-import { CloseIcon, MapTriFold } from "../../icons";
+import {Box, Button, Divider, IconButton, Typography} from "@mui/material";
+import {vars} from "../../theme/variables";
+import {CloseIcon, MapTriFold} from "../../icons";
 import Info from "./Info";
-import { useCdeContext } from "../../CdeContext";
 
-const { baseWhite, gray100, gray700 } = vars
+const {baseWhite, gray100, gray700} = vars
 
 const styles = {
-  root: {
-    padding: '0.75rem 1.5rem',
-    gap: '1.5rem',
-    display: 'flex',
-    alignItems: 'center',
-    borderBottom: `0.0625rem solid ${gray100}`,
-    background: baseWhite,
-    position: 'sticky',
-    zIndex: 9,
-    top: 0,
+    root: {
+        padding: '0.75rem 1.5rem',
+        gap: '1.5rem',
+        display: 'flex',
+        alignItems: 'center',
+        borderBottom: `0.0625rem solid ${gray100}`,
+        background: baseWhite,
+        position: 'sticky',
+        zIndex: 9,
+        top: 0,
 
-    '& .MuiTypography-root': {
-      fontSize: '0.875rem',
-      fontWeight: 600,
-      lineHeight: '142.857%',
-      color: gray700
+        '& .MuiTypography-root': {
+            fontSize: '0.875rem',
+            fontWeight: 600,
+            lineHeight: '142.857%',
+            color: gray700
+        }
     }
-  }
 }
 
-const Header = () => {
-  const { handleClose, setInfoOpen, infoOpen } = useCdeContext();
+const Header = (props: { onClose: () => void, isInfoOpen: boolean, setIsInfoOpen: (b: boolean) => void }) => {
+    const {onClose, isInfoOpen, setIsInfoOpen} = props
```

Comment:
No longer gets the necessary values from the context but rather from props. Remaining changes are from the linter